### ISSUE TITLE
Paying for octobox annually?

### ISSUE THREAD

**Issue by choldgraf** [octobox/octobox#1732]
Hello 👋 this isn't a question about octobox tech itself, but on the process of donating.

*tl;dr - does anybody know if there's a way to pay for a year's worth of octobox instead of month-by-month?*

I saw that it's possible to support octobox "the company", which I'd like to do via a grant that we've got (I work at an educational institution so money flows through grants). However, I couldn't figure out how to pay for more than 1 month at a time. If I could front-load, say, a year of Octobox, then that'd be easier from a reimbursement perspective. Anybody know how I could do that? Just make a one-time donation that doesn't fall under one of the "tiers" of opencollective?

Also as a short side-point, the "gift box" icon points people directly to opencollective, rather than to the page that gives folks the option of one-or-the-other (github marketplace or opencollective). Just wanted to flag it!

**Comment by BenJam** [comment]
hey @choldgraf :wave: thanks for the support! 

I believe you can pay for an annual subscription through the GH marketplace but, if it's easier, you can just email us and we can send you an invoice directly. If you'd prefer to donate to the community directly then you can do so on Open Collective. 

Good point about the gift box!

**Comment by choldgraf** [comment]
is one or the other seen as more beneficial to you two? I'm inclined to support the Ltd, since I like the idea of explicitly signaling that I support a company doing good OS work (and because something tells me that it'll be easier to get reimbursed if it looks like I'm "buying" something), but I can't figure out how to pay for an annual subscription rather than monthly. Will report back if I can figure it out!

**Comment by BenJam** [comment]
The choice is yours but, if you'd prefer to support Andrew and myself specifically then the company is the best way to do that. 

There's definitely an annual plan (2 months free) set up on the GH marketplace btw.

**Comment by choldgraf** [comment]
ok, I will look into this more :-)

**Comment by choldgraf** [comment]
Well, I still cannot figure out how to get something annually in the github marketplace. From the page that's there, this is the only set of options I see:

![image](https://user-images.githubusercontent.com/1839645/56234019-3f206200-6039-11e9-8a8b-00a21affa33f.png)

Apologies if I'm being annoying here - I'd like to support the project, but it's not super easy to figure out how to do so :-(   Maybe this is because it's still on the free trial period?

If I'm being dense and this is noise, then I can close the issue and not worry about it...

**Comment by choldgraf** [comment]
Ah ha! I think I figured it out:

https://help.github.com/en/articles/changing-the-duration-of-your-billing-cycle#changing-the-duration-of-your-personal-accounts-billing-cycle

The setting isn't done from within the github marketplace, but within the user's individual settings.

**Comment by BenJam** [comment]
Cool ta, I'll add some further guidance about this and thanks for the support 👍

**Comment by choldgraf** [comment]
No problem! Here's what I did in case it's helpful:

## How can I pay for Octobox annually instead of monthly?

By default, GitHub Marketplace will charge your account monthly instead of
annually. To pay for [Octobox on the GitHub marketplace](https://github.com/marketplace/octobox)
annually, you need to change this setting in your personal account.
Make sure you are signed in, then go to this link:

https://github.com/settings/billing

Click on the "Change to monthly[annual] billing" link in order to switch
your account from monthly to annual billing. For example, here's a screenshot
of what you'll see after you've switched to annual billing.

![chrome_2019-04-24_13-19-32](https://user-images.githubusercontent.com/1839645/56690983-c0689c00-66ac-11e9-9398-05f755363cca.png)
